.gitignore

```diff
@@ -1,4 +1,5 @@
-
+*.pyc
+*.pyo
```

Comment:
It is much better in python3.2, it generates all .py[co]  in a __pycache__ folder. I guess ignoring pyc is not so much an issue as impacts only tests. Nevertheless it still bothers me to have files generated in the sources during compilation. Anyway, this test suite is ugly, making it a little bit more isn't a big deal. :+1: